src/Routes/SeasonsController.ts

```diff
@@ -404,13 +411,172 @@ export class SeasonsController extends BaseController {
          *             message:
          *               type: string
          */
-        this.router.get("/:seasonId(\\d+)/services/:serviceId(\\d+)/serviceHymns/:serviceHymnId(\\d+)/formats/:formatId(\\d+)", async (req: express.Request, res: express.Response) => {
+        this.router.get(`/:seasonId(\\d+)/${RouteParts.services}/:serviceId(\\d+)/${RouteParts.serviceHymns}/:serviceHymnId(\\d+)/${RouteParts.formats}/:formatId(\\d+)`, async (req: express.Request, res: express.Response) => {
             try {
                 const result = await dataProvider.getServiceHymnFormat(req.params.seasonId, req.params.serviceId, req.params.serviceHymnId, req.params.formatId);
                 res.send(result);
             } catch (ex) {
                 BaseController._OnError(ex, res);
             }
         });
+
+        /**
+         * @swagger
+         *
+         * definitions:
+         *   ServiceHymnFormatContent:
+         *     type: object
+         *     properties:
+         *       id:
+         *         type: integer
+         *       name:
+         *         type: string
+         *       content:
+         *         type: object

```

Comment:
@behoyh, how do I mark this to say that it could be one of the following?
    ITextContent |
    IHazzatContent |
    IVerticalHazzatContent |
    IVideoContent |
    IInformationContent